### PR TITLE
Update changelog for 4.1 release

### PR DESCRIPTION
**Pull request by pascalchevrel**

### CODE REVIEW

File: app/models/changelog_data.php [hunk 1]
```diff
@@ -1,16 +1,19 @@
...
-    'sphinx'     => 'https://github.com/SphinxKnight',
-    'tchevalier' => 'https://github.com/TheoChevalier',
-    'Thegennok'  => 'https://github.com/Thegennok',
+    'Benoit'      => 'https://github.com/benoit-l',
+    'Clement'     => 'https://github.com/clement-lefevre',
+    'flod'        => 'https://github.com/flodolo',
+    'geb'         => 'https://github.com/mgoessen',
+    'jesus'       => 'https://github.com/tx2z',
+    'keko'        => 'https://github.com/keko',
+    'Anthony'     => 'https://github.com/MatonAnthony',
```
Comment (flodolo) [inline app/models/changelog_data.php]: The order of this array is pretty weird, should we order them by displayed name, given it's also easy to do it directly in Atom?

File: app/models/changelog_data.php [hunk 2]
```diff
@@ -48,9 +51,177 @@
...
+            'type'    => ['better'],
+            'issues'  => [609],
+            'authors' => ['Thegennok'],
+        ],
+        [
+            'message' => ['Use <a href="https://github.com/hirak/prestissimo">Prestissimo</a> to parallelize dependency installations on Travis CI'],
+            'section' => ['developers'],
+            'type'    => ['better'],
+            'issues'  => [601],
+            'authors' => ['Antony'],
```
Comment (TheoChevalier) [inline app/models/changelog_data.php]: typo, Anthony

File: app/models/changelog_data.php [hunk 3]
```diff
@@ -48,9 +51,177 @@
...
+        ],
+        [
+            'message' => ['Client-side speed improvement: load a local favicon'],
+            'section' => ['other'],
+            'type'    => ['better'],
+            'issues'  => [575],
+            'authors' => ['Thegennok'],
+        ],
+        [
+            'message' => ['Client-side speed improvement: Transition from woff to woff2 fonts'],
```
Comment (TheoChevalier) [inline app/models/changelog_data.php]: nitpicking: WOFF and WOFF2

**Comment by TheoChevalier** [comment]
I can't spot anything else :beers:

**Pull request merged**